CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
+# Unreleased
+
+- On Windows, fix fullscreen doesn't preserve minimized/maximized state.
```

Comment:
Nit: "fix fullscreen not preserving" is more grammatically correct than "fix fullscreen doesn't preserve".